policy/modules/session/wayland.fc

```diff
@@ -0,0 +1,2 @@
+/run/user/%{USERID}/wayland-[0-9]+	-s	gen_context(system_u:object_r:wayland_runtime_t,s0)
+/run/user/%{USERID}/wayland-[0-9]+\.lock	--	gen_context(system_u:object_r:wayland_runtime_t,s0)
```

Comment:
It may not be super important, but waypipe includes alphabet characters here, too.